OmiseSDK/Configuration.swift

```diff
@@ -0,0 +1,44 @@
+import Foundation
+
+enum Environment {
+    case staging
+    case production
+    
+    private var vaultBaseURL: URL {
+        switch self {
+        case .staging: return URL(string: "[STAGING_URL]")!
+        case .production: return URL(string: "https://vault.omise.co")!
+        }
+    }
+    
+    private var apiBaseURL: URL {
+        switch self {
+        case .staging: return URL(string: "[STAGING_URL]")!
+        case .production: return URL(string: "https://api.omise.co")!
+        }
+    }
+    
+    var tokenURL: URL {
+        return vaultBaseURL.appendingPathComponent("tokens")
+    }
+    
+    var sourceURL: URL {
+        return apiBaseURL.appendingPathComponent("sources")
+    }
+    
+    var capabilityURL: URL {
+        return apiBaseURL.appendingPathComponent("capability")
+    }
+}
+
+struct Configuration {
```

Comment:
What would you think about adding something like this:
```
static let `default` = Configuration()
```
I guess there will be only one configuration so I think as a singleton it would be more convenient to use?

Reply:
I agree with this case. Updated 🙇